kernel/src/main/java/org/kframework/kompile/Kompile.java

```diff
@@ -215,6 +225,20 @@ public CompiledDefinition run(File definitionFile, String mainModuleName, String
         return def;
     }
 
+    private Definition postProcessJSON(Definition defn, String postProcess) {
+    //public K externalParse(String parser, String value, Sort startSymbol, Source source, CompiledDefinition compiledDef) {
+        List<String> command = new ArrayList<>(Arrays.asList(postProcess.split(" ")));
+        Map<String, String> environment = new HashMap<>();
+        RunProcess.ProcessOutput output = RunProcess.execute(environment, files.getProcessBuilder(), command.toArray(new String[command.size()]));
+
```

Comment:
@radumereuta I don't know the proper way to do this. What I want is to pipe the JSON of the defn into this process, and extract the stdout. I don't know how to pipe the JSON of the defn into stdin here.